lib/bolt/executor.rb

```diff
@@ -326,6 +326,18 @@ def wait_until(timeout, retry_interval)
       end
     end
 
+    def prompt(prompt = '', options = {})
+      STDERR.print("#{prompt}: ")
+      value = if options['sensitive']
+                STDIN.noecho(&:gets).chomp
+              else
+                STDIN.gets.chomp
+              end
+      STDERR.puts
```

Comment:
This is only needed in the `sensitive` case. In the normal case, the user's newline will be echoed back to them.